VaderSharp/VaderSharp/SentiText.cs

```diff
@@ -27,20 +28,20 @@ public SentiText(string text)
         private Dictionary<string, string> WordsPlusPunc()
         {
             string noPuncText = Text.RemovePunctuation();
-            var wordsOnly = noPuncText.Split().Where(x=>x.Length > 1);
+            var wordsOnly = Regex.Matches(noPuncText, "\\S{2,}");
```

Comment:
Oh, sorry. I didn't expect it would be slower. I thought LINQ functions are time consuming.